pandas/core/ops.py

```diff
@@ -944,24 +943,20 @@ def na_op(x, y):
             # integer comparisons
 
             # we have a datetime/timedelta and may need to convert
```

Comment:
Is this comment still correct? Or is it only for `y` now?

Reply:
It's only for `y` now.